doc/Administration-guide/Upgrading-your-instance/0.8.1-to-0.9.0/index.md

```diff
@@ -29,7 +29,7 @@ This guide describes how to upgrade your [Kaa server]({{root_url}}Glossary/#kaa-
 Follow these steps to run the upgrade script:
 
 1. Go to your Kaa host machine.
-2. Download the [upgradeKaa081to090.sh](https://github.com/kaaproject/kaa/blob/release-0.10/server/upgrade/data-migration-0.8.1-0.9.0/upgradeKaa081to090.sh), [upgradeMongo081to090KaaVersion.js](https://github.com/kaaproject/kaa/blob/release-0.10/server/upgrade/data-migration-0.8.1-0.9.0/upgradeMongo081to090KaaVersion.js), and [upgradePostgresql081to090kaaVersion.sql](https://github.com/kaaproject/kaa/blob/release-0.10/server/upgrade/data-migration-0.8.1-0.9.0/upgradePostgresql081to090kaaVersion.sql) files.
+2. Download the [upgradeKaa081to090.sh]({{github_url}}server/upgrade/data-migration-0.8.1-0.9.0/upgradeKaa081to090.sh), [upgradeMongo081to090KaaVersion.js]({{github_url}}server/upgrade/data-migration-0.8.1-0.9.0/upgradeMongo081to090KaaVersion.js), and [upgradePostgresql081to090kaaVersion.sql]({{github_url}}server/upgrade/data-migration-0.8.1-0.9.0/upgradePostgresql081to090kaaVersion.sql) files.
```

Comment:
Ok, let's use {{github_url_raw}} here. 
Old links referred to the HTML page on GitHub, not to a file.